nntrainer/tensor/manager.cpp

```diff
@@ -398,7 +398,7 @@ std::vector<Weight *> Manager::requestWeights(
      */
     if (Weight::isGradientClipByGlobalNorm(clip_by_global_norm)) {
       grad_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
-      var_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
+      // var_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
```

Comment:
Is there any reason not to delete the line, but to comment it out?

If this is really unnecessary, leaving it commented will confuse other developers a few months later.

Reply:
I add a comment. Thanks.